build/yaml/jobs/codeanalysis/apiscan.yaml

```diff
@@ -44,15 +48,21 @@ jobs:
         !**\*Test*
       TargetFolder: '$(Build.ArtifactStagingDirectory)\ApiScan'
 
+  # Patch the binary version in the exclusion file.
+  - powershell: |
+      (Get-Content $(APIScanExclusionFilePath)).Replace("[version]", "$(Build.BuildNumber)") | Set-Content $(APIScanExclusionFilePath)
+    displayName: Set binary version in APIScan exclusion list
+
   # Run API Scan
 
-  - task: securedevelopmentteam.vss-secure-development-tools.build-task-apiscan.APIScan@2
+  - task: securedevelopmentteam.vss-secure-development-tools.build-task-apiscan.APIScan@1
```

Comment:
APIScan@2 is fails when providing an exclusion list, that's why I switched to APIScan@1. I'm trying to get help from the task maintainers and see if I can switch back to the higher version.

Reply:
Use the baseline instead of the exclusions. That's why this isn't working. The way that you are expected to exclude false positives is to pass a file to the task that does the build check, not directly to APIScan.

---
In reply to: [522557817](https://github.com/microsoft/CLRInstrumentationEngine/pull/325#discussion_r522557817) [](ancestors = 522557817)